obsreport/obsreport_exporter.go

```diff
@@ -26,28 +26,63 @@ import (
 
 const (
 	// ExporterKey used to identify exporters in metrics and traces.
-	ExporterKey = "exporter"
+	exporterKey = "exporter"
 
 	// SentSpansKey used to track spans sent by exporters.
-	SentSpansKey = "sent_spans"
+	sentSpansKey = "sent_spans"
 	// FailedToSendSpansKey used to track spans that failed to be sent by exporters.
-	FailedToSendSpansKey = "send_failed_spans"
+	failedToSendSpansKey = "send_failed_spans"
 
 	// SentMetricPointsKey used to track metric points sent by exporters.
-	SentMetricPointsKey = "sent_metric_points"
+	sentMetricPointsKey = "sent_metric_points"
 	// FailedToSendMetricPointsKey used to track metric points that failed to be sent by exporters.
-	FailedToSendMetricPointsKey = "send_failed_metric_points"
+	failedToSendMetricPointsKey = "send_failed_metric_points"
 
 	// SentLogRecordsKey used to track logs sent by exporters.
-	SentLogRecordsKey = "sent_log_records"
+	sentLogRecordsKey = "sent_log_records"
 	// FailedToSendLogRecordsKey used to track logs that failed to be sent by exporters.
-	FailedToSendLogRecordsKey = "send_failed_log_records"
+	failedToSendLogRecordsKey = "send_failed_log_records"
 )
 
+// GetExporterKey returns the exporterKey
+func GetExporterKey() string {
```

Comment:
Don't use `Get` as a prefix for accessors.  See https://golang.org/doc/effective_go#Getters.

In this case, this would just be:

```go
func ExporterKey() string {
  return exporterKey
}
```